app/src/main/java/ca/marshallasch/veil/FragmentViewPost.java

```diff
@@ -119,6 +120,17 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         RecyclerView.Adapter recyclerAdapter = new CommentListAdapter(comments);
         recyclerView.setAdapter(recyclerAdapter);
 
+        //click listener for comment bar
+        ImageView commentBar = view.findViewById(R.id.comment_bar);
```

Comment:
why is this an image view?

Reply:
couldn't make a material button to be the style I wanted.